src/nifake/tests/test_session.py

```diff
@@ -328,3 +328,34 @@ def test_multiple_return_params(self):
             assert self.patched_errors.handle_error.call_count == 2
             self.patched_errors.handle_error.assert_called_with(session, self.patched_library.niFake_GetCalDateAndTime.return_value)
     '''
+
```

Comment:
When there are changes to a test file, either system tests or unit tests, there should be a corresponding change in the generated files.

When you don't have the changed generated file as part of your change, it ends up being an unrelated change in a different pull request.